PagerDuty/public-support-scripts
# Enable extensions

## Description
This python script will enable all temporarily disabled extensions on an account. 

I've run into a few customers wanting this;  e.g., for bulk reenabling after resolving a server issue or after returning to normalcy after maintenance.  

### Steps to test changes
* Disable some webhooks (e.g., with a bad URL)
* Check the UI to make sure they're disabled
* Run the script
* Check the UI to make sure they re-enabled

## Documentation 
- [ ] There is documentation that needs to be updated upon merging these changes
